specification/migrate/resource-manager/Microsoft.Migrate/AssessmentProjects/preview/2023-04-01-preview/examples/BusinessCaseAvsSummaryOperations_Get_MaximumSet_Gen.json

```diff
@@ -0,0 +1,243 @@
+              "storageCost": 27,
+              "computeCost": 19,
+              "itLaborCost": 29,
+              "networkCost": 28,
+              "ahubSavings": 1,
+              "securityCost": 4,
+              "managementCostDetails": {
+                "managementCost": 22,
+                "managementCostComponents": [
+                  {
+                    "name": "Unknown",
+                    "value": 16
+                  }
+                ]
+              },
+              "esuSavings": 28
+            },
+            "osLicensingDetails": [
+              {
+                "osType": "Windows",
+                "totalCost": 11,
+                "decomissionCost": 29
+              }
+            ],
+            "distributionByOperatingSystem": [
+              {
+                "osClassificationType": "WindowsOSInSupport",
+                "count": 4
+              }
+            ],
+            "onPremisesUtilizationData": {
+              "numberOfActiveEntities": 10,
+              "numberOfInactiveEntities": 18,
+              "numberOfDecommisionEntities": 10,
+              "numberOfUnknownEntities": 13
+            },
+            "distributionByVirtualization": [
+              {
+                "virtualizationType": "Vmware",
+                "count": 30
+              }
+            ],
+            "iaasOsSupportStatusDistribution": {
+              "unknownSupport": 2,
+              "mainStream": 20,
+              "extended": 26,
+              "outOfSupport": 29
+            },
+            "iaasSqlSupportStatusDistribution": {
+              "unknownSupport": 2,
+              "mainStream": 20,
+              "extended": 26,
+              "outOfSupport": 29
+            },
+            "osServicePackInsight": {
+              "unknownServicePack": 3,
+              "patched": 14,
+              "unpatched": 5
+            },
+            "sqlServicePackInsight": {
+              "unknownServicePack": 3,
+              "patched": 14,
+              "unpatched": 5
+            },
+            "distributionByOperatingSystemVersion": [
+              {
+                "type": "Microsoft Windows Server 2016 Datacenter",
+                "count": 27
+              }
+            ],
+            "distributionBySqlVersion": [
+              {
+                "type": "SQL Server 2012",
+                "count": 27
+              }
+            ],
+            "distributionBySqlEdition": [
+              {
+                "type": "Enterprise Evaluation Edition",
+                "count": 27
+              }
+            ]
+          }
+        },
+        "id": "/subscriptions/ADC896AD-6A38-454E-9A62-AFC618F5F4BC/resourceGroups/rgopenapi/providers/Microsoft.Migrate/assessmentprojects/multipleto8617project/businessCases/sample-business-case/avssummaries/default",
+        "name": "default",
+        "type": "Microsoft.Migrate/assessmentprojects/businessCases/avssummaries",
```

Comment:
>assessmentprojects [](http://example.com/codeflow?start=35&length=18)

casing should be assessmentProjects, not assessmentprojects

Reply:
Done